src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -99,6 +107,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String artistEntryString = request.getParameter("artist"); 
     long timestamp = System.currentTimeMillis();
     String email = currentUser.getEmail();
+    String lyrics;
+
+    // Calls lyrics API
+    Client client = ClientBuilder.newClient();
+    Response lyricsResponce = client.target("https://api.lyrics.ovh/v1/" + artistEntryString + "/" + songEntryString)
+      .request(MediaType.TEXT_PLAIN_TYPE)
+      .get();
+    JsonElement lyricsElement = new JsonParser().parse(lyricsResponce.readEntity(String.class));
+    JsonObject  lyricsObject = lyricsElement.getAsJsonObject();
+    lyrics = (String) lyricsObject.get("lyrics").getAsString().replace("\'","");
```

Comment:
Curious, could you declare the String lyrics here, where you set it equal to the lyrics object, instead of above?

Reply:
Yes, you could. I thought it would be a good idea to declare it above because all of the other variables were declared at the top. Looking back, however, there is probably no real practicality to it.